Sealnote/src/main/java/com/twistedplane/sealnote/data/Note.java

```diff
@@ -102,12 +101,12 @@ public void writeToParcel(Parcel outParcel, int flags) {
         outParcel.writeInt(mId);
         outParcel.writeInt(mPosition);
         outParcel.writeString(mNoteTitle);
+        outParcel.writeString(mType.name());
```

Comment:
Oh alright. You are maintaining the order of read/write in parcel. Looks good then.